### PR TITLE
docs(Search): remove imports of other examples

### PR DESCRIPTION
**Pull request by Fabianopb**
As discussed in #3479 , this PR remove imports in Search examples of other examples. This creates a bit of repetition, which doesn't feel very nice, but all sandboxes work now.

I wonder if it would make sense to add some kind of general test to assure that this won't repeat in other examples... 🤔

### CODE REVIEW

**Comment by levithomason** [comment]
>I wonder if it would make sense to add some kind of general test to assure that this won't repeat in other examples... 🤔

Perhaps.  We have `examples-test.js` which ensures all examples render without errors.  Wonder if we could extend that.

This may also be something we can do with lint rules.  There is an eslint import plugin.  We might be able to restrict imports via an .eslintrc in the doc examples directory.  That would be easier to maintain.

We can merge this for now 👍

File: docs/src/examples/modules/Search/Variations/index.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ const SearchVariationsExamples = () => (
     />
     <ShorthandExample
       title='Input'
-      description='A search can be passed an input via shorthand props.'
+      description='A search can receive an input via shorthand props.'
```
Comment (levithomason) [inline docs/src/examples/modules/Search/Variations/index.js:21]: 👍

**Review by levithomason — APPROVED** [review]

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.86.0`.

**Comment by Fabianopb** [comment]
> 
> This may also be something we can do with lint rules. There is an eslint import plugin. We might be able to restrict imports via an .eslintrc in the doc examples directory. That would be easier to maintain.
> 

@levithomason I've been fiddling with this idea for a while, I can submit a PR soon!

**Pull request merged**